tools/downloader/downloader.py

```
@@ -0,0 +1,188 @@
+            report_list_url: 報告書一覧ページのURL
+            year: 公表年
+        """
+        # 報告書一覧ページを解析してリンクを取得
+        pdf_links = self.page_parser.parse_report_list_page(report_list_url)
+
+        # 各リンクを処理
+        for pdf_link in pdf_links:
+            if isinstance(pdf_link, PdfLink):
+                if self.process_pdf_link(pdf_link, year):
+                    # インターバルを設ける
+                    if not self.dry_run:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)